components/layout_2020/flexbox/layout.rs

```diff
@@ -1941,6 +1941,10 @@ impl FlexItem<'_> {
                             .vec2_to_flow_relative(self.content_max_size)
                             .map(|size| size.map_or(Size::Initial, Size::Numeric)),
                         flex_axis.vec2_to_flow_relative(self.pbm_auto_is_zero),
+                        LogicalVec2 {
+                            inline: Size::FitContent,
+                            block: Size::FitContent,
+                        },
```

Comment:
Nit: since this is repeated a few times, it could be better to add

```rs
impl<T: Copy> From<T> for LogicalVec2<T> {
    fn from(value: T) -> Self {
        Self {
            inline: value,
            block: value,
        }
    }
}
```

in `components/layout_2020/geom.rs`.

And then `Size::FitContent.into()` will suffice.

Reply:
On it!